docs/track.rst

```diff
@@ -568,6 +569,21 @@ If you want as much reproducibility as possible you can choose the `deterministi
 
 If you have more complex needs on how to model traffic, you can also implement a :doc:`custom schedule </adding_tracks>`.
 
+.. _track_ramp_up:
+
+Ramp-up load
+~~~~~~~~~~~~
+
+For benchmarks involving many clients it can be useful to increase load gradually. This avoids load spikes at the beginning of a benchmark when Elasticsearch is not yet warmed up. Rally will gradually add more clients over time but each client will already attempt to reach its specified target throughput. The diagram below shows how clients are added over time:
+
+.. image:: track-ramp-up.png
+   :alt: How ramp-up works
+
+.. note::
+
+    As the number of clients and thus the throughput varies during ramp-up, you should set ``warmup-time-period`` at least to the same value as ``ramp-up-time-period`` to get stable measurement results.
```

Comment:
My first question was "do ramp up and warm up overlap and are they related?" This answers this but not directly. I think we could be more explicit.  Maybe the image tells a thousand words?

Reply:
"warmup-time-period greater than or equal to" I think is clearer? Also, can we see a viable use case where the warm-up is less than the ramp-up? I don't think we want to be still ramping after warmup? If not, can we actually prevent this?

Reply:
Yes, let's prevent this. I've pushed 518d66c0bea9cb7b8c77ddd92ba1b25fb678a46d to enforce this behavior.